assets/component-facets.css

```diff
@@ -744,20 +809,20 @@ details.menu-opening .mobile-facets__close svg {
 }
 
 .mobile-facets__highlight {
-  opacity: 0;
-  visibility: hidden;
-}
-
-.mobile-facets__checkbox:checked + .mobile-facets__highlight {
-  visibility: visible;
-  opacity: 1;
   position: absolute;
   top: 0px;
   left: 0px;
   right: 0px;
   bottom: 0px;
   display: block;
   background-color: rgba(var(--color-foreground), 0.04);
+  opacity: 0;
+  visibility: hidden;
+}
+
+.mobile-facets__checkbox:checked + .mobile-facets__highlight {
+  opacity: 1;
+  visibility: visible;
```

Comment:
Oh I actually forgot that I got rid of highlight completely from the markup for the grid, but what it _was_doing before is it was an invisible element that had `gap` applied because it's now in a display flex with gap. Like I said I got rid of it from my markup later, but I still think it's beneficial to do this because we never want it to take up space, which is what this was doing before.